src/styles/ErrorPage.css

```
@@ -0,0 +1,26 @@
+* {
+   margin: 0;
+   padding: 0;
+}
+
+.errorPage {
+    position: absolute;
+    top: 15%;
+    left: 17%;
+    right: 17%;
+    color: whitesmoke;
+    bottom: 20%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.errorPage_content {
+    max-width: 600px;
+    text-align: center;
+}
+
+.errorPage_content p {
+    font-size: 1.4em;
+    color: gray;
+}
```

Can we use className (err-num and err-text) instead of `style={{}}` for Header in `ErrorPage.js` and add the styles in `ErrorPage.css` as mentioned below?
```suggestion
}

.ui.header.err-num {
    color: #2185D0;
    font-size: 10vw;
    font-weight: bolder;
    line-height: 1em;
    text-transform: uppercase;
}

.ui.header.err-text {
    font-size: 2em;
    margin-bottom: 20px;
    text-transform: uppercase;
    max-width: 600px;
    color: #0d0d0d;
}
```

---

Any specific reason @codesankalp ? Is this a common convention?